.eslintrc.cjs

```diff
@@ -113,7 +113,11 @@ module.exports = {
 		},
 		// Rules for bin and cli files.
 		{
-			files: [ 'bin/**/*.js', 'bin/**/*.mjs', 'packages/cli/src/*.cjs' ],
+			files: [
+				'bin/**/*.js',
+				'bin/**/*.mjs',
+				'**/packages/cli/src/*.cjs',
```

Comment:
```suggestion
			files: [
				'bin/**/*.js',
				'bin/**/*.mjs'
			]
```

Reply:
There are no bin js files, so i removed this entirely